DuckDuckGo/SwipeTabsCoordinator.swift

```diff
@@ -217,6 +226,12 @@ extension SwipeTabsCoordinator: UICollectionViewDelegate {
     }
 
     func scrollViewDidEndDecelerating(_ scrollView: UIScrollView) {
+        guard !state.isIdle else {
+            Pixel.fire(pixel: .swipeTabsIncorrectScrollState)
+            assertionFailure("invalid state")
```

Comment:
Let me know if this is too aggressive. Only devs should see the assertion so it would be good to capture it when it does.

Reply:
286497